src/bindings/python/flux/job/frobnicator/frobnicator.py

```diff
@@ -66,6 +66,7 @@ class JobFrobnicator:
     namespace.
     """
 
+    default_frobnicators = ["defaults", "constraints"]
```

Comment:
commit message nit: "Loads 'defaults' and 'constraints' plugins..." :arrow_right: "Load 'defaults' and 'constraints' plugins..."

Reply:
fixed that and forced a push